client/app/lib/popover_view.coffee

```diff
@@ -28,31 +28,42 @@ module.exports = class PopoverView extends BaseView
             trigger: 'manual'
             title: @titleTemplate @getRenderData()
             html: true
-            placement: @getDirection()
+            placement: @getDirection
```

Comment:
Okay, I didn't realize bootstrap would call the function if it's a function, or use the value if it's not.